### PR TITLE
Notifications Endpoints

### PR DESCRIPTION
**Pull request by colinyoung87**

### CODE REVIEW

File: app/serializers/story_serializer.rb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 class StorySerializer < ActiveModel::Serializer
   attributes :id, :project_id, :team_id, :name, :slug, :description, :owner_id, :created_at, :updated_at, :deleted_at
   has_one :owner
+
+  def owner
```
Comment (phawk) [inline app/serializers/story_serializer.rb]: don’t think this is needed, has_one will include if it exists just

File: spec/api/v1/notifications_controller_spec.rb [hunk 2]
```diff
@@ -1,5 +1,83 @@
 require 'rails_helper'
 
 RSpec.describe "Notifications API", type: :api do
-  it "needs some tests"
+  let!(:user) { create(:user_with_team) }
+  let(:team) { user.teams.first }
+
+  let!(:notification_1) { create(:notification, user: user, team: team, action_performed: "created", seen: false) }
```
Comment (phawk) [inline spec/api/v1/notifications_controller_spec.rb]: there’s a `create_list` factory girl helper that creates a given amount of records for you

**Comment by phawk** [comment]
:+1:

File: app/serializers/story_serializer.rb (hunk 1 above)
Comment (colinyoung87) [inline app/serializers/story_serializer.rb]: it was when the story itself has been archived, and it blows up because it is trying to access `owner` on nil

**Comment by phawk** [comment]
@colinyoung87 this ones good to go just needs updated with ember branch as there are conflicts of some kind

**Pull request merged**